scout/commands/load/research.py

```diff
@@ -122,3 +122,6 @@ def research(case_id, institute, force):
         case_obj["is_research"] = True
         case_obj["research_requested"] = False
         adapter.update_case(case_obj, keep_date=True)
+
+        # Update case variants count
+        adapter.case_variants_count(case_obj["_id"], case_obj["owner"], True)
```

Comment:
👍

Reply:
Oh, but not named arguments so better add a category!
```suggestion
        adapter.case_variants_count(case_obj["_id"], case_obj["owner"], 'research', True)
```

Reply:
(or keep it None and give named for `force_update_case`)

Reply:
Right, thanks!